go.mod

```diff
@@ -26,30 +25,28 @@ require (
 	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
-	go.pinniped.dev/generated/1.19/apis v0.0.0-00010101000000-000000000000
-	go.pinniped.dev/generated/1.19/client v0.0.0-00010101000000-000000000000
+	go.pinniped.dev/generated/1.20/apis v0.0.0-20210107184752-9b8e4f4d5b65
```

Comment:
So, this is partially my bad, because I know we regenerated this file together, but I am plagued by the fact that the version tag was `00010101000000-000000000000 ` previously, and now it is `20210107184752-9b8e4f4d5b65`. I _think_ `20210107184752-9b8e4f4d5b65` means "this version was from 2021, the month of January (`01`), on the 7th day (`07`), at time 6:47:52 (`18`, `47`, `52`), minute, and has checksum `9b8e4f4d5b65`". My gut says that we should flip this tag back to `00010101000000-000000000000`. The reason I think that is 1) because it was like that before and 2) so that we never accidentally pin to an exact version (i.e., we always want the latest version in our codebase).

Happy to video chat about this if you want to.